sections/00_title_authors.md

```diff
@@ -2,58 +2,10 @@
 
 This article is also available as a [preprint on bioRxiv](https://doi.org/10.1101/142760).
 
-Travers Ching<sup>1,*</sup>,
-Daniel S. Himmelstein<sup>2</sup>,
-Brett K. Beaulieu-Jones<sup>3</sup>,
-Alexandr A. Kalinin<sup>4</sup>,
-Brian T. Do<sup>5</sup>,
-Gregory P. Way<sup>2</sup>,
-Enrico Ferrero<sup>6</sup>,
-Paul-Michael Agapow<sup>7</sup>,
-Wei Xie<sup>8</sup>,
-Gail L. Rosen<sup>9</sup>,
-Benjamin J. Lengerich<sup>10</sup>,
-Johnny Israeli<sup>11</sup>,
-Jack Lanchantin<sup>12</sup>,
-Stephen Woloszynek<sup>9</sup>,
-Anne E. Carpenter<sup>13</sup>,
-Avanti Shrikumar<sup>14</sup>,
-Jinbo Xu<sup>15</sup>,
-Evan M. Cofer<sup>16</sup>,
-David J. Harris<sup>17</sup>,
-Dave DeCaprio<sup>18</sup>,
-Yanjun Qi<sup>12</sup>,
-Anshul Kundaje<sup>19</sup>,
-Yifan Peng<sup>20</sup>,
-Laura K. Wiley<sup>21</sup>,
-Marwin H.S. Segler<sup>22</sup>,
-Anthony Gitter<sup>23,†</sup>,
-Casey S. Greene<sup>2,†</sup>
+{{authors.authors}}
```

Comment:
There is a delineation between data and formatting. Data should go in `stats.json` / the stats object. I.e. `authors.affiliations` is a list.

Ideally, all formatting should occur in the markdown source through jinja2. See http://jinja.pocoo.org/docs/2.9/templates/#list-of-control-structures. I think you'll want a for loop.

This may be sort of a pain, but it's important from a design perspective.

Make sense?